test/common/testlib.py

```diff
@@ -1528,6 +1528,8 @@ def arg_parser(enable_sit=True):
                             help="Sit and wait after test failure")
     parser.add_argument('--nonet', dest="fetch", action="store_false",
                         help="Don't go online to download images or data")
+    parser.add_argument('--enable-network', dest='enable_network', action='store_true',
+                        help="Enable network access for tests")
```

Comment:
I'm not sure how useful that's actually going to be -- tests should be able to declare themselves that they need network access, i. e. call `MachineCase.setUp()` accordingly. But it's a quick fix for composer's run-tests invocation indeed, and it doesn't hurt for now.

Reply:
hm no, this is actually only for cockpit's run-tests, which creates the global machine itself.